### PR TITLE
Add note about bot to the issue template

### PR DESCRIPTION
**Pull request by webknjaz**
##### SUMMARY
Better explain users why it's worth filling in issue template.

##### ISSUE TYPE

‒

##### COMPONENT NAME
‒

##### ANSIBLE VERSION
‒

##### ADDITIONAL INFORMATION

‒

### CODE REVIEW

**Comment by webknjaz** [comment]
-label needs_triage

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <!---
 Verify first that your issue/request is not already reported on GitHub.
```
Comment (gundalow) [inline .github/ISSUE_TEMPLATE.md:2]: Maybe add in caps
THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED

Comment (bcoca) [reply .github/ISSUE_TEMPLATE.md:2]: and to be clear, bcoca is not a machine, he just seems like one

Comment (webknjaz) [reply .github/ISSUE_TEMPLATE.md:2]: @gundalow @bcoca Better now?

**Comment by webknjaz** [comment]
-label bug

**Review by gundalow — APPROVED** [review]

**Pull request merged**